tests/plugins/test_store_metadata.py

```diff
@@ -233,15 +252,20 @@ def test_metadata_plugin_rpmqa_failure(tmpdir):
     workflow.prebuild_results = {}
     workflow.postbuild_results = {
         PostBuildRPMqaPlugin.key: RuntimeError(),
+        PLUGIN_KOJI_UPLOAD_PLUGIN_KEY: {'metadata_fragment_key': 'metadata.json',
+                                        'metadata_fragment': 'configmap/build-1-md'}
```

Comment:
Just an observation: both of these are post-build plugins and the rpm-qa one runs before koji-upload. If it fails, koji-upload will not be run at all.

That means this is testing a hypothetical situation in which they are deliberately run the other way around, to test whether the store-metadata plugin can deal with it.

Seems valid.